crates/oracle/src/lib.rs

```diff
@@ -90,6 +90,8 @@ pub mod pallet {
         InvalidOracleSource,
         /// Exchange rate not specified or has expired
         MissingExchangeRate,
+        /// Feeder has already feeded at this block
+        AlreadyFeeded,
```

Comment:
```suggestion
        /// Feeder has already fed at this block
        AlreadyFed,
```
Isn't this the correct conjugation?

Reply:
Taken from https://github.com/open-web3-stack/open-runtime-module-library/blob/9e041dc9d213f843b18b3008f32f3acabb287dcb/oracle/src/lib.rs#L103